File: listas-logpp/lista 2/lista 2 sala/questao 9.py
numero1 = int(input("Digite o primeiro número inteiro: "))
numero2 = int(input("Digite o segundo número inteiro: "))

def eh_divisivel_por_4_ou_5(numero):
    return numero % 4 == 0 or numero % 5 == 0

numeros_divisiveis = []

for numero in (numero1, numero2):
    if eh_divisivel_por_4_ou_5(numero):
        numeros_divisiveis.append(numero)

if numeros_divisiveis:
    print("Os números divisíveis por 4 ou 5 são:", ", ".join(map(str, numeros_divisiveis)))
else:
    print("Nenhum dos números é divisível por 4 ou 5.")

File: listas de exercicio/lista 2/lista 2 sala C/questao 9.c
#include <stdio.h>

int main(){
    int N1, N2;
    printf("Digite o primeiro numero:");
    scanf("%d", &N1);
    printf("Digite o segundo numero:");
    scanf("%d", &N2);

    if (N1 % 4==0 || N1 % 5==0){
        printf("%d é divisivel por 4 e 5, ", N1);
    } if(N2 % 4==0 || N2 % 5==0){
         printf("%d é divisivel por 4 e 5, ", N2);
    }
}